src/home_robot_hw/home_robot_hw/env/stretch_pick_and_place_env.py

```diff
@@ -59,9 +65,12 @@ def __init__(
                 sem_gpu_id=0,
             )
 
-        # Create a simple grasp planner object, which will let us pick stuff up.
-        # This takes in a reference to the robot client - will replace "self" with "self.client"
-        self.grasp_planner = GraspPlanner(self, visualize_planner=visualize_planner)
+        if ros_grasping:
+            # Create a simple grasp planner object, which will let us pick stuff up.
+            # This takes in a reference to the robot client - will replace "self" with "self.client"
+            self.grasp_planner = GraspPlanner(self, visualize_planner=visualize_planner)
+        else:
+            self.grasp_planner = None
```

Comment:
Can we add a check here which explicitly says that "`visualize_planner` will do nothing as no planner is created" if `ros_grasping==False` but `visualize_planner==True`.

Reply:
done, will throw error